sys/include/net/gnrc/sixlowpan/ctx.h

```diff
@@ -113,7 +114,7 @@ gnrc_sixlowpan_ctx_t *gnrc_sixlowpan_ctx_update(uint8_t id, const ipv6_addr_t *p
                                                 uint8_t prefix_len, uint16_t ltime,
                                                 bool comp);
 
-#ifdef MODULE_GNRC_SIXLOWPAN_CTX
+#if IS_USED(MODULE_GNRC_SIXLOWPAN_CTX) || DOXYGEN
```

Comment:
Additionally, you can do some optional cleanup if you want and move this `#ifdef` as a C-conditional into `gnrc_sixlowpan_ctx_remove()` ;-).

Reply:
I still don't quite understand why this single function was 'protected' this way :man_shrugging: